submissions/evgenii-del/dom/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/style.css">
+    <link rel="shortcut icon" type="image/x-icon" href="./img/favicon.ico">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
+    <title>Kottans side-menu</title>
+    <script src="https://kit.fontawesome.com/b6d85b8ecc.js" crossorigin="anonymous"></script>
```

Comment:
Using external scripts for icons is ok for this task, but avoid using it in the future if you have only 5 icons. Using SVG icons will be better